docs/developer/testing/TESTING.md

```diff
@@ -56,6 +56,10 @@ OPENSHIFT_CI=false \
 make test-e2e
 ```
 
+### OpenStack setup
+
+* no changes needed, just setup s3 storage in aws.
+
```

Comment:
Add info here
https://github.com/openshift/oadp-operator/blob/510c2033cbbae5cbd207da04d99490bd42c41466/docs/developer/testing/TESTING.md?plain=1#L11

not in a new section

Reply:
ok.. added OpenStack in the section you mentioned as well but also kept the openshtack section to make it more obvious.